http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -676,56 +677,6 @@ class LeoPubsubMessageSubscriberSpec
     res.unsafeRunSync()
   }
 
-  it should "create an app with no disk" in isolatedDbTest {
-    val savedCluster1 = makeKubeCluster(1).save()
-    val savedNodepool1 = makeNodepool(1, savedCluster1.id).save()
-
-    val savedApp1 = makeApp(1, savedNodepool1.id).save()
-
-    val assertions = for {
-      getAppOpt <- KubernetesServiceDbQueries
-        .getActiveFullAppByName(savedCluster1.googleProject, savedApp1.appName)
-        .transaction
-      getApp = getAppOpt.get
-    } yield {
-      getApp.app.errors shouldBe List()
-      getApp.cluster.asyncFields shouldBe Some(
-        KubernetesClusterAsyncFields(IP("1.2.3.4"),
-                                     IP("0.0.0.0"),
-                                     NetworkFields(Config.vpcConfig.networkName,
-                                                   Config.vpcConfig.subnetworkName,
-                                                   Config.vpcConfig.subnetworkIpRange))
-      )
-      getApp.app.appResources.disk shouldBe None
-      getApp.app.status shouldBe AppStatus.Running
-      getApp.app.appResources.kubernetesServiceAccountName shouldBe Some(
-        ServiceAccountName("gxy-ksa")
-      )
-    }
-
-    val res = for {
-      tr <- traceId.ask
-      dummyNodepool = savedCluster1.nodepools.filter(_.isDefault).head
-      msg = CreateAppMessage(
-        savedCluster1.googleProject,
-        Some(ClusterNodepoolAction.CreateClusterAndNodepool(savedCluster1.id, dummyNodepool.id, savedNodepool1.id)),
-        savedApp1.id,
-        savedApp1.appName,
-        None,
-        Map.empty,
-        AppType.Galaxy,
-        Some(tr)
-      )
-      queue <- InspectableQueue.bounded[IO, Task[IO]](10)
-      leoSubscriber = makeLeoSubscriber(asyncTaskQueue = queue)
-      asyncTaskProcessor = AsyncTaskProcessor(AsyncTaskProcessor.Config(10, 10), queue)
-      _ <- leoSubscriber.handleCreateAppMessage(msg)
-      _ <- withInfiniteStream(asyncTaskProcessor.process, assertions)
-    } yield ()
-
-    res.unsafeRunSync()
-  }
-
   it should "be able to create an multiple apps in a cluster" in isolatedDbTest {
```

Comment:
Not part of your changes but might as well fix a typo while at it 😛 
```suggestion
  it should "be able to create multiple apps in a cluster" in isolatedDbTest {
```